src/TestJwt.java

```diff
@@ -0,0 +1,20 @@
+import com.auth0.jwt.*;
+import com.auth0.jwt.algorithms.*;
+import com.auth0.jwt.exceptions.*;
+import java.io.UnsupportedEncodingException;
+
+
+public class TestJwt {
+
+	public static void main(String args[]) {
+		try {
+    			String token = JWT.create()
+        		.withIssuer("auth0")
```

Comment:
Yes the issuer needs to be a CA - need to get a valid CA for the production version